manifests/prometheus/alerts.d/diego_cell_mem_few_candidates.yml

```diff
@@ -12,7 +12,17 @@
           severity: critical
         annotations:
           summary: Few candidate cells for a 4GiB app
-          description: Only {{ $value | printf "%.0f" }} cell(s) are advertising enough free memory to run a 4GiB app instance. Some tenant app(s) may be scaling out of control or we may have depleted cell capacity.
+          description: |
+            Only {{ $value | printf "%.0f" }} cell(s) are advertising enough free memory to run a 4GiB app instance. The threshold is 3 (1 per AZ).
+
+            The likely impact is some tenants are unable to schedule some apps, but failures may be sporadic.
+
+            It means we don't have enough capacity to run all tenant workloads. This is probably caused by natural growth, but it is also possible
+            that one or more tenant applications are scaling out beyond their intended limtis (e.g. because of a bug in the autoscaler, or a fault
+            in the tenant's autoscaling policy).
+
+            Consider adding more cells, or finding and scaling down the offending application.
```

Comment:
I would emphasize the latter over the former. If something's managed to get to this point I would assume it would swallow more cells worth of memory within a matter of hours

Reply:
Mm. Will swap the order of operations there. Something like "check for something consuming all the resources it can gets its hands on, then scale if it's not that"

Reply:
Addressed